tools/protocol-schema-check/src/main.rs

```diff
@@ -94,6 +92,8 @@ fn compute_type_hash(
 const PROTOCOL_SCHEMA_FILE: &str = "protocol_schema.toml";
 
 fn main() {
+    let _: EpochInfoAggregator = Default::default();
```

Comment:
What's this? (Only sanity checking that you intended to include that line here)

Reply:
Workaround to include one struct which is not included automatically (I don't know why).
Actually I found a better way to express this issue and will proceed with it.